plugins/modules/mssql_script.py

```diff
@@ -62,6 +62,20 @@
     type: bool
     default: false
     version_added: 8.4.0
+  encryption:
+    description:
+      - Specify whether to use encryption for the connection to the server.
+        Please refer to the pymssql documentation for detailed information.
+      - The available values are "off" and "on"
```

Comment:
```suggestion
      - The available values are V("off") and V("on").
```
(I left the quotes to avoid conversion to booleans.)

If no other values are allowed, how about adding `choices` to limit the input to these two values?